src/main/java/org/myrobotlab/swing/ProgramABGui.java

```diff
@@ -40,9 +40,9 @@
   static final String START_SESSION_LABEL = "Start Session";
   // TODO: make this auto-resize when added to gui..
   private JTextField text = new JTextField("", 30);
-  private JTextArea response = new JTextArea("BOT Response :");
+  private JEditorPane response = new JEditorPane("text/html", "BOT Response :");
+  String textAreacontent = "";

```

Comment:
This variable shouldn't be here. please remove this line.  This should be local to the method that uses it  (it's not something that should exist in the global context.. because thats "response.getText()"  not this variable ...